index.html

```diff
@@ -320,10 +337,11 @@ <h3 class="skills_dropdown_title dropdown_child_class">
                       </li>
                     </ul>
                   </li>
-               </ul>
+              </ul>
+             </div>
            </section>
 
-           <section class="contact" id="contact">
+           <section class="contact" id="Contact">
```

Comment:
[Optional]
- It would be better if you center your form to improve your design and make professional.
- your design:
<img width="373" alt="image" src="https://user-images.githubusercontent.com/77190203/149167158-80ed9034-370f-4d85-b47d-1a989e6c6bd8.png">